backend/forms/models/form.py

```diff
@@ -89,5 +90,11 @@ def validated_fields(self):
         return [Field.model_validate(field_dict) for field_dict in self.enriched_fields]
 
     @property
-    def survey(self):
-        return self.event.surveys.filter(languages=self).get()
+    def survey(self) -> Optional["Survey"]:
+        from .survey import Survey
+
+        # there can only be one
+        try:
+            return self.event.surveys.filter(languages=self).get()
+        except Survey.DoesNotExist:
+            return None
```

Comment:
```suggestion
        # there can only be one
        return self.event.surveys.filter(languages=self).first()
```
(unless you mean to raise when there are multiple)

Reply:
`Survey.MultipleObjectsReturned` is a realistic failure scenario I want to make visible.